contracts/tokenWhitelist.sol

```diff
@@ -29,15 +30,18 @@ interface ITokenWhitelist {
     function getStablecoinInfo() external view returns (string memory, uint256, uint256, bool, bool, bool, uint256);
     function tokenAddressArray() external view returns (address[] memory);
     function redeemableTokens() external view returns (address[] memory);
-    function updateTokenRate(address, uint, uint) external;
+    function methodIdWhitelist(uint32) external view returns (bool);
```

Comment:
We have _isMethodIdWhitelisted (i can make it _isMethodIdSupported) which is inherited by inheriting TokenWhitelistable.
I followed the general approach where the getters have the same name as the variables (e.g. tokenAddressArray)
By the way all the variables that are private and have a getter should be maybe public instead, since we agreed that we follow this pattern for contract that can be inherited so we don't want to expose every variable directly, but TokenWhitelist is a stand alone contract behind ENS. Any thoughts?

Reply:
I am happy with the approach re: getters, as that is what we do everywhere else. 

I think this method should start with `is` like all of the other methods we have that return `boolean` values.